cleanlab/experimental/keras.py

```diff
@@ -32,28 +32,86 @@
 
 import tensorflow as tf
 import numpy as np
+from typing import Callable
 
 
-class KerasWrapper:
+class KerasWrapperModel:
     """
-    KerasWrapper is instantiated in the same way as a ``tf.keras.models.Sequential`` object,
+    KerasWrapperModel takes in a callable function to instantiate a Keras model (using Keras functional API)
+    that is compatible with :py:meth:`CleanLearning.fit()<cleanlab.classification.CleanLearning.fit>`.
+    """
+
```

Comment:
Add a short docstring to describe some of the args below, especially the ones that are not standard Keras Model() init args. Also, a one-line real code example of what `model` might look like could be helpful